src/ProjectTemplates/test/ProjectTemplates.Tests.Common/Infrastructure/GenerateTestProps.targets

```diff
@@ -0,0 +1,94 @@
+<Project>
+  <Target
+    Name="GenerateTestProps"
+    BeforeTargets="CoreCompile"
+    DependsOnTargets="PrepareForTest"
+    Condition="$(DesignTimeBuild) != true">
```

Comment:
Why exclude during a design time build? Normally it's good to run code gen so VS intellisense works correctly.

Reply:
No idea. That's pre-existing code and I'm trying to make the minimal change to get this in for preview5.